src/services/ipfs/ipfs.resolver.ts

```diff
@@ -0,0 +1,19 @@
+import { Inject } from '@nestjs/common';
+import { Args, Mutation, Resolver } from '@nestjs/graphql';
+import { GraphQLUpload } from 'apollo-server-express';
+import { IpfsService } from './ipfs.service';
+
+@Resolver()
+export class IpfsResolver {
+  constructor(@Inject(IpfsService) private ipfsService: IpfsService) {}
+
+  @Mutation(() => String, {
+    description: 'Uploads a file to IPFS endpoint',
+  })
+  async addFileToIpfs(
```

Comment:
In my opinion should be more generic. Like UploadFile/Image. Where is stored should be matter of configuration.